methcomp/mountain.py

```diff
@@ -0,0 +1,242 @@
+    ) -> matplotlib.axes.Axes:
+        """Plot mountain plot
+
+        Parameters
+        ----------
+        xlabel : str, optional
+            The label which is added to the X-axis. (default: "Method difference")
+        ylabel : str, optional
+            The label which is added to the Y-axis. (default: "Folded CDF (%)")
+        label : str, optional
+            mountaint line legend label (default:"Method 1 - Method 2" )
+        unit : str, optional
+            unit to disply for x-axis
+        title : str, optional
+            figure title, if none there will be no title
+            (default: None)
+        color : str, optional
+            Color for mountain plot elements
+        show_hline: bool, optional
+            If set show horizontal lines for iqr
+        show_vlines: bool, optional
+            If set show vertical lines at iqr and median
+        show_markers: bool, optional
+            If set show markers at iqr and median
+        ax : matplotlib.axes.Axes, optional
+            matplotlib axis object, if not passed, uses gca()
+
+        Returns
+        -------
+        matplotlib.axes.Axes
+            axes object with the plot
+        """
+        ax = ax or plt.gca()
+        ax.step(
+            y=self.result["mountain"],
+            x=self.result["quantile"],
+            where="mid",
+            label=f"{label} AUC={self.result['auc']:.2f}",
+            color=color,
+        )
+        if show_hline:
+            ax.hlines(
+                self.result["mountain_iqr"],
+                xmin=self.result["iqr"][0],
+                xmax=self.result["iqr"][1],
+                color=color,
+            )
+        if show_vlines:
+            ax.vlines(
+                self.result["median"],
+                ymin=0,
+                ymax=50,
+                label=f"median={self.result['median']:.2f} {unit or ''}",
+                linestyle="--",
+                color=color,
+            )
+            if self.iqr > 0:
+                ax.vlines(
+                    self.result["iqr"],
+                    ymin=0,
+                    ymax=50 - self.iqr / 2,
+                    label=f"{self.iqr:.2f}% IQR ={self.result['iqr'][1]-self.result['iqr'][0]:.2f} {unit or ''}",
+                    linestyle=":",
+                    color=color,
+                )
+        if show_markers:
+            ax.plot(
+                self.result["median"],
+                self.result["mountain_median"],
+                "o",
+                color=color,
+            )
+            if self.iqr > 0:
+                ax.plot(
+                    self.result["iqr"],
+                    self.result["mountain_iqr"],
+                    "o",
+                    color=color,
+                )
+        u = f"({unit})" if unit else ""
+        ax.set(xlabel=f"{xlabel} {u}", ylabel=ylabel or None, title=title or None)
+        ax.legend(loc="upper left", fontsize="medium")
```

Comment:
can you adjust the legend to be similar to the other plots? 
this means we should add `frameon=False` to the legend call

Reply:
Agree